### PR TITLE
Remove all the special stuff for SPA/Microfrontends. It's now all in the deb.

### PR DESCRIPTION
**Pull request by brandones**
https://pihemr.atlassian.net/browse/UHM-5276

### CODE REVIEW

File: mirebalais-modules/openmrs/manifests/init.pp [hunk 1]
```diff
@@ -214,10 +209,10 @@
...
   }
 
-   # this is legacy, this is now handled by our custom app loader factor
-   file { "/home/${tomcat}/.OpenMRS/appframework-config.json":
-	ensure => absent
-   }
+  # this is legacy, this is now handled by our custom app loader factor
+  file { "/home/${tomcat}/.OpenMRS/appframework-config.json":
+    ensure => absent
+  }
```
Comment (brandones) [inline mirebalais-modules/openmrs/manifests/init.pp:215]: Unrelated indentation fix

**Review by mseaton — APPROVED** [review]
This makes me happy, nice.  Defer to @mogoodrich  on whether there are any specific issues, but glad to see this evolving.

**Comment by mogoodrich** [comment]
Yeah, agreed, nice to have all this removed.
